Makefile

```diff
@@ -1,386 +1,9 @@
-.PHONY: api-gen
-
-cfgdoc-gen:
-	$(GOCC) run ./node/config/cfgdocgen > ./node/config/doc_gen.go
-
-appimage: lotus
-	rm -rf appimage-builder-cache || true
-	rm AppDir/io.filecoin.lotus.desktop || true
-	rm AppDir/icon.svg || true
-	rm Appdir/AppRun || true
-	mkdir -p AppDir/usr/bin
-	cp ./lotus AppDir/usr/bin/
-	appimage-builder
-
-docsgen: docsgen-md docsgen-openrpc fiximports
-
-docsgen-md-bin: api-gen actors-gen
-	$(GOCC) build $(GOFLAGS) -o docgen-md ./api/docgen/cmd
-docsgen-openrpc-bin: api-gen actors-gen
-	$(GOCC) build $(GOFLAGS) -o docgen-openrpc ./api/docgen-openrpc/cmd
-
-docsgen-md: docsgen-md-full docsgen-md-storage docsgen-md-worker
-
-docsgen-md-full: docsgen-md-bin
-	./docgen-md "api/api_full.go" "FullNode" "api" "./api" > documentation/en/api-v1-unstable-methods.md
-	./docgen-md "api/v0api/full.go" "FullNode" "v0api" "./api/v0api" > documentation/en/api-v0-methods.md
-docsgen-md-storage: docsgen-md-bin
-	./docgen-md "api/api_storage.go" "StorageMiner" "api" "./api" > documentation/en/api-v0-methods-miner.md
-docsgen-md-worker: docsgen-md-bin
-	./docgen-md "api/api_worker.go" "Worker" "api" "./api" > documentation/en/api-v0-methods-worker.md
-
-docsgen-openrpc: docsgen-openrpc-full docsgen-openrpc-storage docsgen-openrpc-worker docsgen-openrpc-gateway
-
-docsgen-openrpc-full: docsgen-openrpc-bin
-	./docgen-openrpc "api/api_full.go" "FullNode" "api" "./api" > build/openrpc/full.json
-docsgen-openrpc-storage: docsgen-openrpc-bin
-	./docgen-openrpc "api/api_storage.go" "StorageMiner" "api" "./api" > build/openrpc/miner.json
-docsgen-openrpc-worker: docsgen-openrpc-bin
-	./docgen-openrpc "api/api_worker.go" "Worker" "api" "./api" > build/openrpc/worker.json
-docsgen-openrpc-gateway: docsgen-openrpc-bin
-	./docgen-openrpc "api/api_gateway.go" "Gateway" "api" "./api" > build/openrpc/gateway.json
-
-.PHONY: docsgen docsgen-md-bin docsgen-openrpc-bin
-
-fiximports:
-	$(GOCC) run ./scripts/fiximports
-
-gen: actors-code-gen type-gen cfgdoc-gen docsgen api-gen
-	$(GOCC) run ./scripts/fiximports
-	@echo ">>> IF YOU'VE MODIFIED THE CLI OR CONFIG, REMEMBER TO ALSO RUN 'make docsgen-cli'"
-.PHONY: gen
-
-jen: gen
-
-snap: lotus lotus-miner lotus-worker
-	snapcraft
-	# snapcraft upload ./lotus_*.snap
-
-# separate from gen because it needs binaries
-docsgen-cli: lotus lotus-miner lotus-worker
-	python3 ./scripts/generate-lotus-cli.py
-	./lotus config default > documentation/en/default-lotus-config.toml
-	./lotus-miner config default > documentation/en/default-lotus-miner-config.toml
-.PHONY: docsgen-cli
-
-print-%:
-	@echo $*=$($*)
+%:
+	@echo -e "### ⚠️⚠️⚠️ NOTICE ⚠️⚠️⚠️ ###\n" \
+	         "The 'releases' branch has been deprecated with the 202408 split of 'Lotus Node' and 'Lotus Miner'.\n" \
+		     "See https://github.com/filecoin-project/lotus/blob/master/LOTUS_RELEASE_FLOW.md#why-is-the-releases-branch-deprecated-and-what-are-alternatives for more info and alternatives.\n" \
+		     "----------------------" | tee /dev/stderr
 
+.PHONY: % 
```

Comment:
```suggestion
%:
	@echo -e "### ⚠️⚠️⚠️ NOTICE ⚠️⚠️⚠️ ###\n" \
	         "The 'releases' branch has been deprecated with the 202408 split of 'Lotus Node' and 'Lotus Miner'.\n" \
		     "See https://github.com/filecoin-project/lotus/blob/master/LOTUS_RELEASE_FLOW.md for more info and alternatives.\n" \
		     "----------------------" | tee /dev/stderr
	@exit 1

.PHONY: % 
```

Reply:
After doing some testing, it seems good to have the makefile exit with a non-zero status code to force a very simple upgrade script to fail, making it more likely that the user will notice the issue and see the message.

When running a script that does a "upgrade in-place" routine it will now exit at the `make clean` step:

```
$make clean
### ⚠️⚠️⚠️ NOTICE ⚠️⚠️⚠️ ###
 The 'releases' branch has been deprecated with the 202408 split of 'Lotus Node' and 'Lotus Miner'.
 See https://github.com/filecoin-project/lotus/blob/master/LOTUS_RELEASE_FLOW.md#why-is-the-releases-branch-deprecated-and-what-are-alternatives for more info and alternatives.
 ----------------------
### ⚠️⚠️⚠️ NOTICE ⚠️⚠️⚠️ ###
 The 'releases' branch has been deprecated with the 202408 split of 'Lotus Node' and 'Lotus Miner'.
 See https://github.com/filecoin-project/lotus/blob/master/LOTUS_RELEASE_FLOW.md#why-is-the-releases-branch-deprecated-and-what-are-alternatives for more info and alternatives.
 ----------------------
make: *** [clean] Error 1
```